daemon/call_interfaces.c

```diff
@@ -1738,6 +1738,17 @@ static void call_ng_main_flags(sdp_ng_flags *out, str *key, bencode_item_t *valu
 		case CSH_LOOKUP("recording file"):
 			out->recording_file = s;
 			break;
+		case CSH_LOOKUP("recording-media-slot-offer"):
+			// This needs to be > 0
+			out->media_rec_slot_offer = bencode_get_integer_str(value, out->media_rec_slot_offer);
+			break;
+		case CSH_LOOKUP("recording-media-slot-answer"):
+			// This needs to be > 0
+			out->media_rec_slot_answer = bencode_get_integer_str(value, out->media_rec_slot_answer);
+			break;
```

Comment:
So if the second argument is set to 1 then the default value (if used) would always come out at 1. Then there would need to be some check (perhaps directly within the case statement?) to check/set that the value was at least 1 (and output an error/warning log if the check happens)? Or is there a more elegant/standard place to do this?

Reply:
> So if the second argument is set to 1 then the default value (if used) would always come out at 1. Then there would need to be some check (perhaps directly within the case statement?) to check/set that the value was at least 1 (and output an error/warning log if the check happens)? Or is there a more elegant/standard place to do this?

Using the "previous value" as second argument is fine. Using an explicit `1` as default value doesn't actually solve the problem, as it would still leave the value as zero if either the key/value isn't given at all, or if an explicit zero was given as value. So the code needs to explicitly check this. The most appropriate place to check if the value is sane is the place where it's being read or used, for example in `setup_stream_proc`